### ISSUE TITLE
Accumulator Balancing

### ISSUE THREAD

**Issue by Skillabstinenz** [Dysoch/DyTech#402]
Why use anything else than the normal mk1? The small and big ones just cost more and store less. The higher ranks are just absurdly overpriced, needing at least 5 accus of the lower rank plus other materials, but the gain is laughable. MK1 to MK4 just doubles the storage and I/O, but it needs 25 MK1 accus + lots of other stuff.

One should only need 1 accu of the previous tier and a small buff in their stats could make them worthwhile.

Or make it like the ultimate mining drill, it costs a fortune but you probably only need one anyway.

**Comment by Quizer85** [comment]
Agreed, this and the solar panel variations don't seem very well balanced. Not only the material cost to upgrade them, but also the cost of making the small and large variations. Those are less effective per square occupied than the medium-sized version, yet they cost more in materials.

BTW, you should call the accumulators with different focus by different names. Keep calling the regular ones 'Basic Accumulator', but call the Mk III (which charges faster, but releases power more slowly) 'Quick-Charge Accumulator' or something, and give it the same Mk number of the Basic Accumulator with the same capacity. The Mk II could be called 'Sturdy Accumulator', since all it gets over its predecessor is more HP, but it probably should be rebalanced anyway.
